docs/src/api.md

```diff
@@ -285,9 +285,12 @@ Pathfinding.HeightMap
 Building a custom metric is straightforward, if the provided ones do not suit your purpose.
 See the [Developer Docs](@ref) for details.
 
-## AgentsIO
+## (De)Serialization
 ```@docs
 AgentsIO
 AgentsIO.populate_from_csv!
-AgentsIO.dump_to_csv
+AgentsIO.save_checkpoint
+AgentsIO.load_checkpoint
+AgentsIO.to_serializable
+AgentsIO.from_serializable
```

Comment:
I don't see why these two functions should be exported and discussed in the user API. Perhaps we should discuss them in the developer's docs. But the user shouldn't be using them.